content/en/docs/release_notes/ikeja.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Ikeja release notes
+linkTitle: Ikeja
+description: 11 March 2022
+date: 2022-03-11
+Hide_readingtime: true
+---
+<!-- ## Summary -->
+
+{{% releasenotes/upgrade %}}
+
+<!-- ## Breaking changes -->
+
+## Features
```

Comment:
**issue:** All these new features/parameters as of this comment are missing from the flow-trigger documentation.

Reply:
The trigger documentation is being rewritten in [RDPP-7218](https://jira.axway.com/browse/RDPP-7218).